### DISCUSSION TITLE
ERROR: rg is nil

### DISCUSSION THREAD

**Discussion by seantw1974** [filecoin-project/lotus#8434] [General Q&A]
~# lotus-miner proving check 0
ERROR: rg is nil

but use lotus-miner proving check --slow 0 
is OK

WHY??

**Comment by seantw1974** [comment]
2022-04-05T14:21:13.676+0800	WARN	rpc	go-jsonrpc@v0.1.5/handler.go:279	error in RPC call to 'Filecoin.CheckProvable': rg is nil:

**Reply by seantw1974** [reply]
This happen since yesterday ...

**Reply by seantw1974** [reply]
Seems new version have such issue .
Daemon:  1.15.2-dev+mainnet+git.1bd0f5594.dirty+api1.5.0
Local: lotus-miner version 1.15.2-dev+mainnet+git.1bd0f5594.dirty

**Reply by seantw1974** [reply]
lotus-miner proving check 
can only use "--slow" option now 
Please check

**Comment by TippyFlitsUK** [comment]
Hi @seantw1974 

I have tried to replicate this error using `1.15.1-rc4` on both mainnet and calibration but not seeing the same error.

Can you please post or attch your output for `lotus-miner proving check --slow 0`?

Many thanks!

**Comment by seantw1974** [comment]
Use —slow is ok no problem
Without —slow has such issue

TippyFlits ***@***.***>於 2022年4月5日 週二，下午6:18寫道：

**Reply by TippyFlitsUK** [reply]
Can you please post or attach your output for `lotus-miner proving check --slow 0`?

**Comment by seantw1974** [comment]
After window post success .....the lotus-miner proving check --slow 0  result is changed ???? Why got this error ???

./lotus-miner proving check --slow 0 > out.txt
root@tim-System-Product-Name:~/lotus# cat out.txt
deadline  partition  sector  status
0         0          751     bad (getting commR: No state for /751: datastore: key not found)
0         0          933     bad (getting commR: No state for /933: datastore: key not found)
0         0          1057    bad (getting commR: No state for /1057: datastore: key not found)
0         0          1276    bad (getting commR: No state for /1276: datastore: key not found)
0         0          1622    bad (getting commR: No state for /1622: datastore: key not found)
0         0          92      bad (getting commR: No state for /92: datastore: key not found)
0         0          154     bad (getting commR: No state for /154: datastore: key not found)
0         0          658     bad (getting commR: No state for /658: datastore: key not found)

**Comment by TippyFlitsUK** [comment]
This is starting to look like storage issues @seantw1974.

Can you please check that you have enough available storage/sealing space?

**Reply by seantw1974** [reply]
This actor ID already stop to seal 
I only keep the storage in static mode..
not sealing for a period of time !!
Yesterday the miner All sectors r fail !!!
So I move to new version lotus ...
so I do lotus-miner proving checking --slow 0 ( all passed)
today passed the windows post !!
but after that .....the lotus-miner proving cheking --slow 0 got error log ???????

**Reply by seantw1974** [reply]
Maybe it's RAID card issue ..I will check it ..
Thank you!!

**Comment by TippyFlitsUK** [comment]
Even if you are not sealing, your available storage space will still be reduced by things like the daemon chain etc. Please check that your storage for both sealing and long-term sector storage is stable and available.
